tutorials/continuous-aggs-tutorial.md

```diff
@@ -156,10 +156,6 @@ aggregates only for the data that satisfies the condition:
 `time_bucket('1h', pickup_datetime) < max(pickup_time) - '1h'` (if the
 `refresh_lag` is set to 1 hour)
 
-Both `refresh_lag` or `refresh_interval` can be set to an integer
-value, which is then interpreted as the number of microseconds for the
-lag or interval respectively.
-
 To keep the continuous aggregate up-to-date,
 you can set `timescaledb.refresh_lag` to a negative value. This will allow the continuous aggregate
```

Comment:
This part needs to be revisited.
1) I believe we no longer need a negative value to keep this up to date. Using a value of 0 should suffice. But needs a test before updating the docs. (Since 1.6 the lag is calculated against the  current time as returned by now() )
2) Also, materialized_only setting will help with this. 
Perhaps the best approach is to update this section along with the materialized_only changes.

Reply:
I removed the paragraph. We are describing the effect of setting negative values in `refresh_lag` in the users part of the manual. Since the continuous aggregate is now update to date I do not think we need to describe this "trick" in the tutorial.